src/sinks/datadog/traces/stats.rs

```diff
@@ -18,6 +19,12 @@ const TAG_STATUS_CODE: &str = "http.status_code";
 const TAG_SYNTHETICS: &str = "synthetics";
 const TOP_LEVEL_KEY: &str = "_top_level";
 
+/// The duration of time in nanoseconds that a bucket covers.
+const BUCKET_DURATION_NANOSECONDS: u64 = 10_000_000_000;
```

Comment:
I'd suggest using `Duration` directly (it can be `const` constructed) so that we don't have to mentally juggle the nanoseconds/seconds conversion.

Reply:
It's getting messy to adapt to that ... for the following reasons:
- only std::time::Duration is const construct-able (not chrono::Duration)
- I'm not seeing a way to do the `btime` calculation (bucket window) which currently uses modulo , without converting back to nanos and re-constructing a chrono::Duration.
- The resulting Stats payload data structure we send out requires the start and duration in u64 nanoseconds.

Unless you want to pair on that (happy to), I'll leave it at is.

Reply:
Ah right, stupid `chrono`.

Yeah, fine to leave as-is.